composer.json

```diff
@@ -26,8 +26,9 @@
     "ingenerator/kohana-extras": "^2.0",
     "ingenerator/kohana-core": "^4.7",
     "ingenerator/tokenista": "^1.4",
-    "swiftmailer/swiftmailer": "^6.3",
-    "composer/installers": "^1.9"
+    "composer/installers": "^1.9",
+    "symfony/mailer": "^6.3",
+    "egulias/email-validator": "^3.2.1"
```

Comment:
Can we tackle this by updating the "conflict": definition below? Just to avoid having our own hard dependency on email-validator in case symfony / other things that are actually using it are able to bump to e.g. ^4 in future?

Reply:
Thanks, hadn't spotted that was a thing